src/content_api/details_utils.py

```diff
@@ -0,0 +1,163 @@
+import json
+from collections import OrderedDict
+
+from bs4 import BeautifulSoup
+from lxml import etree
+from lxml import html
+from pandas.io.json import json_normalize
+
+look = ['title', 'body']
```

Comment:
I find it helpful when reading others' code if constants are in all-caps like this https://www.python.org/dev/peps/pep-0008/#constants (it helps me differentiate between things being passed to the function and constants that are set for the whole module), but I guess this is something we can debate as a team

Reply:
Done

Reply:
my java origins jumped out and went with static vars instead of constants for the commit msg, apols